### PR TITLE
7648 accessibility map card 5 2

### PR DESCRIPTION
**Pull request by phudson-he**
### Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change
Changes have been done to ensure [WCAG 2.1 requirements](https://www.w3.org/TR/WCAG21/#requirements-for-wcag-2-1) have been met for accessibility guidelines.

Details of the changes are as follows:

- views/components/workbench.htm : No change
- views/components/map-popup.htm : Added aria-labels and made kb operable
- views/components/map-editor.htm : Added aria-labels and made kb operable
- views/components/cards/map-popup.htm : Added aria-label + ensured kb operable
- views/components/cards/map.htm : Associated labels with fields with id's

### Issues Solved
Ensures that components (listed above) conform to accessibility guidelines
#

### Checklist

-   [x] Unit tests pass locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

#### Ticket Background
*   Sponsored by: Historic England
*   Found by: @phudson-he
*   Tested by: @phudson-he
*   Designed by: @phudson-he

### CODE REVIEW

**Comment by phudson-he** [comment]
@robgaston - This has now had stable/5.2x merged in so now includes accessibility mode/css. Tests and checks all passed.

**Comment by csmith-he** [comment]
@robgaston @aarongundel I've merged in dev/6.1.x into the branch relating to this pull request.  The build appears to be failing on unit tests which are not any of the files changed as part of this pull request.

Please can you advise on next steps with this?

File: arches/app/media/css/arches.scss [hunk 1]
```diff
@@ -2258,17 +2258,13 @@ div.final-cons-step-separator>h4 {
 
 .workbench-card-sidepanel-header-container {
     border-bottom: 1px solid #ddd;
-    padding: 17px 30px 10px 0px;
+    padding: 10px 20px 10px 5px;
```
Comment (aarongundel) [inline arches/app/media/css/arches.scss:2261]: The padding here seems to be off now; there's less below the bottom border than above; these should be equal.  It also looks like there's a slight shift in alignment to the right; the "x" icon should align with the left side of the border start.

**Review by aarongundel — CHANGES_REQUESTED** [review]
Looks pretty good, just a few small items to discuss (looks like there's an outstanding merge conflict also).

File: arches/app/templates/views/components/map-editor.htm [hunk 2]
```diff
@@ -6,7 +6,7 @@
     toggleTab('editor');
 }, css: {
     'active': activeTab() === 'editor'
-}">
+}" tabindex="0" onkeyup="if(event.which == 13 || event.keyCode == 13){ $(this).trigger('click'); }" aria-label="Edit geometry information">
```
[the remark this answers is not in the document]
Comment (phudson-he) [reply arches/app/templates/views/components/map-editor.htm]: keyup and click events now in VM

File: arches/app/templates/views/components/map-editor.htm [hunk 3]
```diff
@@ -318,25 +321,25 @@ <h5 data-bind="text: card.model.instructions"></h5>
...
                     </div>
                     <div id="add-basemap-wizard-help-content">
                         <div>
                             <div class="panel-heading">
-                                <h3 class="panel-title help-panel-title" style="">
+                                <h3 class="panel-title help-panel-title">
                                     <span data-bind="html: card.model.get('helptitle')"></span>
                                 </h3>
                             </div>
-                            <div class="panel-body" style="padding: 10px 10px 15px 10px;" data-bind="html: card.model.get('helptext')">
```
[the remark this answers is not in the document]
Comment (phudson-he) [reply arches/app/templates/views/components/map-editor.htm:339]: Thanks, more to come I promise :)

File: arches/app/media/css/arches.scss (hunk 1 above)
Comment (phudson-he) [reply arches/app/media/css/arches.scss]: Yes you are correct, not sure why this was done previously. Changed back to the previous values.

File: arches/app/media/css/arches.scss [hunk 4]
```diff
@@ -2258,17 +2258,13 @@ div.final-cons-step-separator>h4 {
...
     margin-left: 0px;
     margin-top: -16px;
-    position: fixed;
```
[the remark this answers is not in the document]
Comment (phudson-he) [reply arches/app/media/css/arches.scss:2264]: I think its would be best to keep as a fixed position. Had to add a width so the underlining continues across the panel. Will revisit these fixed width panels in the near future to make them responsive.

**Comment by phudson-he** [comment]
@aarongundel Hi Aaron, all requested changes have been actioned, are you happy to progress? Many thanks.

**Comment by chiatt** [comment]
Hey @phudson-he, sorry for the merge conflicts. I had to get rid of the fixed position to resolve a different problem.

**Pull request closed without merging**